src/main/view/View.cpp

```diff
@@ -181,6 +184,12 @@ void View::onEvent(Event& event) {
 			onEntityDeleted(entityDeletedEvent);
 			break;
 		}
+		case PLAYER_DESTROYED_EVENT: {
+			PlayerDestroyedEvent& (playerDestroyed) =  (PlayerDestroyedEvent&) event;
```

Comment:
So far we only have Entity Created, Moved and Deleted (3 of 4 CRUD). Can we use the Entity Deleted for this instead of creating a new event?